fastlane_bot/events/exchanges/base.py

```diff
@@ -51,6 +52,9 @@ def get_pools(self) -> List[Pool]:
         """
         return list(self.pools.values())
 
+    def get_event_contract(self, w3: Union[Web3, AsyncWeb3]) -> Union[Contract, AsyncContract]:
+        return w3.eth.contract(abi=self.get_abi())
```

Comment:
Can add `.events` at the end of this line, and simply the rest of the code.
Don't forget ro fix the function name and return-value type, if you choose to do it this way.

Reply:
Purpose of this method is to return a particular contract, not just events. It would be too small of a scope in terms of reusability if it only returned events.

Reply:
Yeah, I'm suggesting to replace this function with a `get_events` function, since we don't seem to need the one currently implemented.